src/react-queries/index.ts

```diff
@@ -18,6 +31,102 @@ import { thorchainLp } from 'pages/ThorChainLP/queries/queries'
 export const thorchainBlockTimeSeconds = '6.1'
 const thorchainBlockTimeMs = bn(thorchainBlockTimeSeconds).times(1000).toNumber()
 
+const common = createQueryKeys('common', {
+  allowanceCryptoBaseUnit: (
+    assetId: AssetId | undefined,
+    spender: string | undefined,
+    from: string | undefined,
+  ) => ({
+    queryKey: ['allowanceCryptoBaseUnit', assetId, spender, from],
+    queryFn: async () => {
+      if (!assetId) throw new Error('assetId is required')
+      if (!spender) throw new Error('spender is required')
+      if (!from) throw new Error('from address is required')
+
+      if (!isToken(fromAssetId(assetId).assetReference)) return null
+      const supportedEvmChainIds = getSupportedEvmChainIds()
+      if (!supportedEvmChainIds.includes(fromAssetId(assetId).chainId as KnownChainIds)) return null
```

Comment:
can we just kill KnownChainIds already 😅

Reply:
Absolutely agree with this, this makes no sense - If we have a `ChainId` type, it *is* a "known" ChainId